tests/test_client.py

```diff
@@ -501,13 +486,3 @@ def test_message_update(client: Union[LocalClient, RESTClient], agent: AgentStat
 def test_organization(client: RESTClient):
     if isinstance(client, LocalClient):
         pytest.skip("Skipping test_organization because LocalClient does not support organizations")
-
-
-def test_model_configs(client: Union[LocalClient, RESTClient]):
```

Comment:
can we replace this with a test that tests `client.list_llm_configs()` and `client.list_embedding_configs()`?

Reply:
done